src/components/Game/PhraseScrambler/PhraseScramblerPresentation.js

```diff
@@ -130,9 +130,7 @@ function PhraseScramblerPresentation({
               </div>
               <div
                 data-testid="card-footer"
-                className={`flex flex-row bg-gray-50 px-4 py-4 sm:px-6 ${
-                  !gameCompleted ? 'justify-end' : 'justify-center'
-                }`}
```

Comment:
Is this change in justification intentional?

Reply:
yes, it was off-kilter before. I'm not sure why it was styled that way